### PR TITLE
Add 'relengapi repl' for debugging use

### PR DESCRIPTION
**Pull request by djmitche**
I found this useful, and it doesn't hurt.

### CODE REVIEW

File: relengapi/blueprints/base/__init__.py [hunk 1]
```diff
@@ -85,3 +85,23 @@ def make_parser(self, subparsers):
...
+        return parser
+
+    def run(self, parser, args):
+        import code
+        # try to get readline, but don't fail if it's not available
+        try:
+            import readline
+            assert readline
+        except ImportError:
+            readline = None
```
Comment (Callek) [inline relengapi/blueprints/base/__init__.py]: ...but where is readline module/var used?

Comment (djmitche) [inline relengapi/blueprints/base/__init__.py]: The interpreter uses it if it's present

**Comment by djmitche** [comment]
comment updated

File: relengapi/blueprints/base/__init__.py (hunk 1 above)
Comment (Callek) [inline relengapi/blueprints/base/__init__.py]: Ahh my missing piece was the fact that readline does this with input (and raw_input) https://docs.python.org/2/library/functions.html?highlight=input#input

**Comment by Callek** [comment]
:+1: I've done this with flask's debug mode enabled and simply raising an exception during a request. But having this for cases where I don't need a request context will be useful

**Pull request merged**